### PR TITLE
[51] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Oge-nna**
## Task: `[51] Develop a Contact Form Page using HTML`
- `Created a contact page & restored missing folder'
contactForm

### CODE REVIEW

File: submissions/ogennaOgbogu/HTML Forms/contactForm.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            
+            <br>
+
+            <label for="email">Email</label>
+            <input type="email" name="email" id="email" placeholder="Enter Email here"/>
+
+            <br><br>
+
+            <label for="subject">Subject:</label><br>
+            <textarea name="subject" id="subject" cols="30" rows="10">
```
Comment (adaugochi) [inline submissions/ogennaOgbogu/HTML Forms/contactForm.html:25]: make `subject` an `input` field instead

File: submissions/ogennaOgbogu/HTML Forms/contactForm.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+            </textarea>
+
+            <label for="message">Message:</label><br>
+            <textarea name="message" id="message" rows= "10" cols = "30"> 
+
+            </textarea><br><br>
+
```
Comment (adaugochi) [inline submissions/ogennaOgbogu/HTML Forms/contactForm.html:33]: remove too many newline spaces

File: submissions/ogennaOgbogu/HTML Forms/contactForm.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        
+            <input type="submit" value="submit">
+            <br><br>
+
```
Comment (adaugochi) [inline submissions/ogennaOgbogu/HTML Forms/contactForm.html:37]: same here

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/ogennaOgbogu/HTML Forms/contactForm.html (hunk 2 above)
Comment (Oge-nna) [reply submissions/ogennaOgbogu/HTML Forms/contactForm.html:33]: Okay, thank you for your suggestions, effecting them now.

**Pull request closed without merging**